onvm/go.sh

```diff
@@ -36,6 +36,11 @@ nf_cores=$3
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 verbosity=1
+#default base virtual address set
+virt_addr="--base-virtaddr=0x7f000000000"
+
```

Comment:
style nit: remove lines 41-43

Reply:
Thanks Dennis. I'll make the necessary changes.

Reply:
cool. in addition, i think that we also need to tell the user what the default base virt addr is on startup, and that they can override it if necessary.

Reply:
Ill add this soon.